app/konnect/mesh-manager/federate-zone.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Federate a zone control plane to {{site.konnect_saas}}
+content_type: tutorial
+---
+
+If you already have a zone control plane which is not connected to any global control plane, you can federate it to {{site.konnect_short_name}} in [Mesh Manager](/konnect/mesh-manager/). 
+
+By federating a zone control plane, you move {{site.mesh_product_name}} from a single-zone setup to [multi-zone](/mesh/latest/production/deployment/multi-zone/) setup. One benefit of moving from single-zone to multi-zone is it gives you automatic service fail-over in case a specific zone fails.
+
+This guide explains how to federate a zone control plane to {{site.konnect_short_name}} by migrating an existing {{site.mesh_product_name}} zone to {{site.konnect_short_name}} and adding another zone.
+
+## Prerequisites
+
+* A universal or Kubernetes cluster with a running zone control plane that isn't connected to the global control plane in {{site.konnect_short_name}}
+* [The latest version of `kumactl`](/mesh/latest/production/install-kumactl/) installed and configured to communicate with the self-managed global control plane
+* A [{{site.mesh_product_name}} global control plane in {{site.konnect_short_name}}](/konnect/mesh-manager/service-mesh/#create-a-zone-in-the-global-control-plane)
+
+## Transfer resources from the zone control plane to {{site.konnect_short_name}}
+
+1. In {% konnect_icon mesh-manager %} [**Mesh Manager**](https://cloud.konghq.com/mesh-manager), click **Global Control Plane Actions** and then click **Configure kumactl**.
+
+1. Configure kumactl with the zone control plane:
+
+  1. Kubernetes only: If your zone is deployed on Kubernetes, you can port-forward 5681 for easy access:
+  
+  ```sh
+  kubectl port-forward svc/kong-mesh-control-plane -n kong-mesh-system 5681
+  ```
+
+  1. Configure `kumactl` with the admin token:
```

Comment:
@jakubdyszkiewicz Where do you get the admin token? Was that in a previous step?